.travis.yml

```diff
@@ -1,5 +1,13 @@
 ---
 language: ruby
-rvm: 2.1.2
-before_script: bundle exec appraisal install
-script: bundle exec appraisal rspec
+rvm:
+  - 2.0.0
+  - 2.1.8
```

Comment:
2.1.10 is the latest 2.1.x. I wouldn't be opposed to dropping 2.1 completely, though, if others felt it were unneeded. It's only getting security fixes at this point and I think internally we're at 2.2 or 2.3 for everything.